source/type/type_utils.h

```diff
@@ -28,68 +28,22 @@ IRC #navitia on freenode
 www.navitia.io
 */
 
-#include "type/datetime.h"
-#include "type/meta_data.h"
-#include "type/type.pb.h"
 #include <boost/date_time/posix_time/posix_time.hpp>
-#include "type/message.h"
 
 namespace nt = navitia::type;
```

Comment:
We don't want to propagate namespace aliases (in bare .h it is the case)
So we'll have to explicit namespaces here, and
```
#include "type/type.h"
#include "type/type.pb.h"
```